extensions-ecom.js

```
@@ -0,0 +1,489 @@
+        )
+          .then((response) => {
+            if (response.ok) {
+              return response.json()
+            } else {
+              throw new Error('Upload failed: ' + response.statusText)
+              window.voiceflow.chat.interact({
+                type: 'error',
+                payload: {
+                  id: 0,
+                },
+              })
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)